prow/deck/jobs/jobs.go

```diff
@@ -46,13 +46,6 @@ var (
 // TODO(#5216): Remove this, and all associated machinery.
```

Comment:
We wanted to remove this, not entrench it more -- can we do that?

Reply:
I will look into it and see what I can do.

Reply:
The main reason I'm not doing this on the client is that we aren't running unit tests for our Typescript code yet (https://github.com/kubernetes/test-infra/issues/9478), and I really think this should be unit tested. I'm also not very optimistic about the possibility of `deck/jobs` getting removed after > 1 year (#5216), seeing as there's the risk of increasing Deck's bandwidth and memory usage but minimal reward (basically just improved code hygiene, and that only if it's done carefully).